packages/voorbeeld-design-tokens/documentation/link/nl-link.stories.tsx

```diff
@@ -0,0 +1,95 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { Link } from '@nl-design-system-candidate/link-react/css';
+
+const meta = {
+  id: 'nl-link',
+  title: 'Components/Link/Candidate',
+  component: Link,
+  args: {
+    children: 'Read more',
```

Comment:
Ik zou dit Nederlands houden
```suggestion
    children: 'Lees meer',
```